src/Libraries/RevitNodes/Elements/ModelCurve.cs

```diff
@@ -157,9 +159,10 @@ private bool InternalSetSketchPlaneFromCurve(Autodesk.Revit.DB.Curve c)
 
             TransactionManager.Instance.TransactionTaskDone();
 
-            return !needsRemake;
```

Comment:
I think we should not invert this bool, since you already inverted the bool on line 82